test/unit_area.jl

```diff
@@ -0,0 +1,21 @@
+# not implemented for dimension >= 3
+p = BallInf(zeros(3), 1.0)
+@test_throws ArgumentError area(p)
+
+# sets with zero area
+p = Singleton([0.0, 1.0])
+@test area(p) ≈ 0.0
+p = Interval(0.0, 1.0) × Interval(0.0, 0.0)
+@test area(p) ≈ 0.0
+
+# triangle
+p = VPolygon([[13, 14.], [16, 30.], [50, 10.]])
+@test area(p) ≈ 302.0
+
+# quadrilateral
+vlist = [[3, 4.], [5, 11.], [12, 8.], [9, 5.], [5, 6.]]
+p = VPolygon(vlist) # convex hull => 4 points
+@test area(p) ≈ 35.0
+
+# general case (non-convex)
+@test LazySets._area_2D(vlist) ≈ 30.0
```

Comment:
What does "non-convex" mean?
I would rather test the `area` method for the case `m > 4` instead of the helper method.

Reply:
the area function applies to non-convex polygons too, and this example here is the one in wikipedia: see "More Complex Example" in https://en.wikipedia.org/wiki/Shoelace_formula

Reply:
but if you have another polygon example to test i can do add it, sure.

Reply:
>  instead of the helper method.

note that i tested the helper alone because `VPolygon(..)` would take the convex hull (unless i ask it not to do so..)

Reply:
Okay, I understand now. Thanks for the explanation.